CHANGELOG.md

```diff
@@ -4,8 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][Keep a Changelog] and this project adheres to [Semantic Versioning][Semantic Versioning].
 
+<!-- markdownlint-disable MD024 -->
 ## [Unreleased]
 
+### Changed
+
+- Refactor : crate/module document move to pure Markdown ([#136](https://github.com/SpringQL/SpringQL/pull/136))
```

Comment:
[nits] I interpreted from "keep a changelog" and semantic versioning that `Changed` sections include "breaking changes", which bump up major versions.

But "keep a changelog" is a human readable format (not a machine readable one) and you noted `Refactor:`, it should be OK to add this line in CHANGELOG.